systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -94,7 +95,13 @@ void testUserWithNameMoreThan64Chars(ExtensionContext extensionContext) {
 
         final Condition condition = KafkaUserResource.kafkaUserClient().inNamespace(Environment.TEST_SUITE_NAMESPACE).withName(userWithLongName).get().getStatus().getConditions().get(0);
 
-        verifyCRStatusCondition(condition, "only up to 64 characters", "ExecutionException", "True", NotReady);
+        assertThat(condition.getStatus(), CoreMatchers.is("True"));
+        assertThat(condition.getType(), CoreMatchers.is("NotReady"));
```

Comment:
We have enums for these two

Reply:
IIRC there is also for the `True` -> we are using it in `ResourceManager` f.e.

Reply:
Awesome, thanks